taxi/views.py

```diff
@@ -1,47 +1,53 @@
+from django.contrib.auth.decorators import login_required
```

Comment:
The import statement for `login_required` is correct, but ensure that it is used consistently across all function-based views that require authentication.